libsolidity/analysis/DocStringAnalyser.cpp

```diff
@@ -36,21 +36,51 @@ using namespace solidity::frontend;
 namespace
 {
 
-void copyMissingTags(StructurallyDocumentedAnnotation& _target, set<CallableDeclaration const*> const& _baseFunctions)
+void copyMissingTags(set<CallableDeclaration const*> const& _baseFunctions, StructurallyDocumentedAnnotation& _target, CallableDeclaration const* _declaration = nullptr)
 {
 	if (_baseFunctions.size() != 1)
 		return;
 
 	auto& sourceDoc = dynamic_cast<StructurallyDocumentedAnnotation const&>((*_baseFunctions.begin())->annotation());
```

Comment:
Are you only using the first element of a pointer-based set? Can this cause nondeterminism?

Reply:
I am, but it's a set of the size() == 1 as you can see in line 41. 
I am mainly passing a set to avoid at least 4 lines of casting and extracting `_baseFunctions.begin()` before calling the function. I figured I need to do that several times, I might as well do it in the function.